shelf/bulk_update/utils.py

```diff
@@ -1,10 +1,9 @@
 import logging
-import os
 import sys
 import shelf.configure as configure
 from shelf.bulk_update.container import Container
-from shelf.bulk_update.index_pruner import IndexPruner
-from shelf import background_utils
+from shelf.bucket_update.utils import update_search_index
+from shelf.bucket_update.utils import prune_search_index
```

Comment:
Just a suggestion but there are a few ways you could do this instead. Here are a couple

```
from shelf.bucket_update import utils
...
create_runner(utils.update_search_index)
create_runner(utils.prune_search_index)
```

or

```
from shelf.bucket_update.utils import update_search_index, prune_search_index
```